forem/forem
# Hide the buttons on the tag page

## What type of PR is this? (check all applicable)


## Description
Follow and Hide buttons for the tag page.

## Related Tickets & Documents



- Related Issue #https://github.com/forem/forem/issues/20022
- Closes #20022 

## QA Instructions, Screenshots, Recordings

https://www.loom.com/share/6d71195191734665afd8dbe71293758e?sid=5c7484a3-d0e8-4cc9-822a-8f070f0f6b38

Click on the "Follow" and "Hide" button on any of the tag pages `/t/<tag_name>` and make sure they are actually been hidden and followed by going to the /tags page or the dashboard.

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/jQDozgWeDXUoQZ1htF/giphy.gif)
